README.md

```diff
@@ -888,21 +895,42 @@ models:
 Currently, this package uses different variables to adapt the models to your objectives and naming conventions. They can all be updated directly in `dbt_project.yml`
 
 <details>
-<summary><b>Coverage Variables</b></summary>
+<summary><b>Testing and Documentation Variables</b></summary>
 
 | variable    | description | default     |
 | ----------- | ----------- | ----------- |
 | `test_coverage_pct` | the minimum acceptable test coverage percentage | 100% |
 | `documentation_coverage_pct` | the minimum acceptable documentation coverage percentage | 100% |
+| `primary_key_test_macros` | the set(s) of test macro names used to test primary key macros | [["dbt.test_unique", "dbt.test_not_null"], "dbt_utils.test_unique_combination_of_columns"] |
```

Comment:
I'm confused by this "the set(s) of test macro names used to test primary key macros" - can we update it to something a bit clearer. Maybe something like "the set(s) of dbt tests used to check validity of a primary key"